deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -0,0 +1,110 @@
+from deepchem.feat.base_classes import MolecularFeaturizer
+from deepchem.utils.typing import RDKitMol
+from deepchem.utils.molecule_feature_utils import one_hot_encode
+import numpy as np
+from rdkit import Chem
+from sklearn.metrics import pairwise_distances
+
+
+class MATFeaturizer(MolecularFeaturizer):
+  """
+  This class is a featurizer for the Molecule Attention Transformer [1]_.
+  The featurizer accepts an RDKit Molecule, and a boolean (one_hot_formal_charge) as arguments.
+  The returned value is a numpy array which consists of molecular graph descriptions:
+    - Node Features
+    - Adjacency Matrix
+    - Distance Matrix
+
+  References
+  ---------
+  .. [1] Lukasz Maziarka et al. "Molecule Attention Transformer`<https://arxiv.org/abs/2002.08264>`"
+
+  Examples
+  --------
+  >>> import deepchem as dc
+  >>> feat = dc.feat.MATFeaturizer()
+  >>> out = feat.featurize("CCC")
+  """
+
+  def __init__(
+      self,
+      one_hot_formal_charge: bool = True,
+  ):
+    """
+    Parameters
+    ----------
+    add_dummy_node: bool, default True
```

Comment:
Remove `add_dummy_node` from the docstring too!

Reply:
Done!